pennylane_lightning/core/_serialize.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, device_name, use_csingle: bool = False):
         self.hamiltonian_c64 = lightning_ops.observables.HamiltonianC64
         self.hamiltonian_c128 = lightning_ops.observables.HamiltonianC128
 
+        self.use_mpi = False
```

Comment:
Adding to @Lee suggestion:
```suggestion
        self._use_mpi = use_mpi
```

Reply:
Nice catch